src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -315,6 +325,191 @@ def get_item_name(self, item: Any) -> Any:
                         "{} doesn't contain a metadata.name. Unable to track changes".format(
                             self.api_func))
 
+class DefaultCreator():
+    def __init__(self, inventory: 'Dict[str, List[Device]]', coreV1_api: 'client.CoreV1Api', storage_class: 'str'):
+        self.coreV1_api = coreV1_api
+        self.storage_class = storage_class
+        self.inventory = inventory
+
+    def parse_drive_group_size(self, drive_group_size: Optional[str]) -> Tuple[int, int]:
```

Comment:
Note we have that code already in ceph: https://github.com/ceph/ceph/blob/master/src/python-common/ceph/deployment/drive_group.py